tests/test_struct/test_struct_graph.py

```diff
@@ -201,7 +201,7 @@ def test_add_node_with_variant(self):
         graph = BELGraph()
         namespace, name, identifier, variant_name = n(), n(), n(), n()
         node = protein(namespace=namespace, name=name, identifier=identifier, variants=hgvs(variant_name))
-        parent = node.get_parent()
```

Comment:
Same here - I think the point of this is just to test that the function doesn't throw an error. We could also check that it's not none and an instance of `pybel.dsl.BaseEntity`.